frontend/apps/docs/source.config.ts

```diff
@@ -13,6 +12,6 @@ export default defineConfig({
         dark: 'github-dark-high-contrast',
       },
     },
-    remarkPlugins: [remarkInstall],
+    remarkPlugins: [],
```

Comment:
Apparently, remark-install has been removed and switched to remarkNpm.
https://github.com/fuma-nama/fumadocs/releases/tag/fumadocs-docgen%402.1.0